src/config.ts

```diff
@@ -8,6 +8,7 @@ export default {
     'guyz',
     'guyzz',
     'guyss',
+    'guys',
```

Comment:
Thank you for your contribution, but this is already covered by the library we use https://github.com/retextjs/retext-equality/blob/main/rules.md

Please see comments on lines 4,5,7 above

Reply:
Oh okay, I didn't know about it. 

Have a great day, Eddie! 🙂